background.ts

```diff
@@ -355,6 +355,15 @@ Electron.ipcMain.on('k8s-integration-set', async(event, name, newState) => {
   }
 });
 
+/** The first time this event is called the data arrives somewhat slowly.
+ *  Shuffle the names so the order varies each time (not by much with only 3 items,
+ *  but as we add more there will be more variation).
+ *  Note that the sorting method is correct -- calling `Math.random` in the actual
+ *  sort comparison function gives bogus results (not that we really care here).
+ */
+Electron.ipcMain.on('k8s-integration-extra-info', (event) => {
+  k8smanager.listIntegrationWarnings(event).then();
```

Comment:
```suggestion
  k8smanager.listIntegrationWarnings(event);
```

Reply:
Yeah, this just fell out of changes discussed above.